<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Streaming</class>
 <widget class="QWidget" name="Streaming">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_enroll">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>410</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ENROLL</string>
   </property>
  </widget>
  <widget class="QFrame" name="streamingFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="streamingLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 300 bold 13pt &quot;Quicksand Light&quot;;
color:  rgb(243,115,33);
background:none;
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="text">
     <string>Streaming Video</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="bestshotFrame">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>311</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 13pt &quot;Quicksand Medium&quot;;
color:  rgb(243,115,33);
background:none;
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:none;
border:1px solid rgb(255, 255, 255);
font: 300 bold 13pt &quot;Quicksand Light&quot;;
color:  rgb(243,115,33);
background:none;</string>
     </property>
     <property name="text">
      <string>Detected Vehicle Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:none;
border:1px solid rgb(255, 255, 255);
font: 300 bold 13pt &quot;Quicksand Light&quot;;
color:  rgb(243,115,33);
background:none;</string>
     </property>
     <property name="text">
      <string>Vehicle1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="dateTimeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 500 10pt &quot;Quicksand Medium&quot;;
background-color: rgba(243,115,33,0.5);
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="text">
     <string>Date - Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="licensetitleLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 500 10pt &quot;Quicksand Medium&quot;;
background-color: rgba(243,115,33,0.5);
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="text">
     <string>LicensePlate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="bestshotFrame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(243,115,33);
background:none;
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <zorder>streamingLabel</zorder>
   <zorder>dateTimeLabel</zorder>
   <zorder>licensetitleLabel</zorder>
   <zorder>bestshotFrame</zorder>
   <zorder>bestshotFrame_2</zorder>
  </widget>
  <widget class="QPushButton" name="pushButton_network">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>360</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>NETWORK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_connect">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>390</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>CONNECT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>310</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>0000~00000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_url">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>280</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>___.___.___.___</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>250</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;IPv4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>310</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;PORT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
